src/icons/safari-pinned-tab.svg

```diff
@@ -0,0 +1,29 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="628.000000pt" height="628.000000pt" viewBox="0 0 628.000000 628.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.11, written by Peter Selinger 2001-2013
+</metadata>
+<g transform="translate(0.000000,628.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M2700 6180 c-129 -21 -354 -57 -500 -81 -236 -38 -267 -45 -288 -66
+-26 -26 -26 -14 -6 -180 6 -47 5 -51 -12 -47 -11 2 -205 33 -432 69 -378 59
+-416 63 -442 50 -15 -8 -31 -22 -34 -31 -4 -10 -32 -175 -62 -368 -31 -193
+-59 -354 -62 -357 -4 -3 -38 -14 -77 -23 -403 -99 -643 -398 -737 -916 -26
+-145 -36 -543 -19 -738 42 -476 188 -1101 353 -1517 99 -248 274 -547 448
+-765 108 -134 325 -352 458 -458 419 -334 894 -547 1422 -636 833 -141 1689
+80 2380 614 115 89 391 363 484 480 360 454 581 983 652 1561 20 162 23 507 5
+659 -29 249 -76 455 -155 685 -129 377 -300 662 -537 895 l-97 95 27 390 c17
+238 24 400 19 415 -17 52 7 49 -834 106 -138 9 -263 14 -277 10 -46 -11 -55
+-39 -70 -226 l-14 -175 1 249 1 249 -25 18 c-23 16 -75 19 -541 35 -284 9
+-528 14 -543 10 -44 -11 -56 -56 -57 -208 -1 -239 -10 -240 -47 -8 -29 187
+-34 207 -57 228 -33 28 -39 28 -325 -18z m678 -2566 c111 -23 214 -69 302
+-135 72 -54 130 -115 130 -137 0 -5 -22 -30 -50 -56 -65 -61 -84 -60 -162 10
+-120 109 -223 154 -348 156 -116 1 -199 -41 -239 -121 -70 -136 -16 -223 217
+-357 219 -126 290 -184 340 -279 24 -44 27 -61 27 -145 0 -86 -3 -101 -32
+-162 -89 -189 -307 -276 -558 -223 -141 30 -266 96 -354 189 -28 30 -51 59
+-51 63 0 18 88 103 107 103 11 0 38 -14 59 -30 145 -112 236 -150 358 -150
+140 0 231 75 231 190 0 96 -50 143 -310 294 -76 44 -160 102 -189 131 -100 96
+-134 208 -102 333 43 170 153 282 319 327 70 19 211 18 305 -1z"/>
+</g>
+</svg>
```

Comment:
Maybe try to proccess over svgo?

`npx svgo -i src/icons/safari-pinned-tab.svg`